Makefile

```diff
@@ -78,7 +78,7 @@ docker-build: build
 	docker build . -t ovsdb-etcd -f dist/images/Dockerfile.ovsdb-etcd
 
 .PHONY: docker-push
-docker-push:
+docker-push: docker-build
```

Comment:
no ... this is intentional ... to enable independent running and debugging of each part ... docker-build will run currently even if image is up to date

Reply:
OK, I added another common rule